includes/admin/class-amp-post-meta-box.php

```diff
@@ -170,23 +178,32 @@ public function render_status( $post ) {
 			is_post_type_viewable( $post->post_type )
 			&&
 			current_user_can( 'edit_post', $post->ID )
-			&&
-			! amp_is_canonical()
 		);
 
 		if ( true !== $verify ) {
 			return;
 		}
 
-		$errors = AMP_Post_Type_Support::get_support_errors( $post );
-		$status = post_supports_amp( $post ) ? self::ENABLED_STATUS : self::DISABLED_STATUS;
+		if ( current_theme_supports( 'amp' ) ) {
```

Comment:
Would it be possible having a comment for this `if-then-else`?

Reply:
To explain why the condition is here?